docker-compose.yml

```diff
@@ -6,14 +6,15 @@ services:
     image: agrdocker/agr_file_generator:latest
     volumes:
       - type: volume
-        source: agr_data_share
-        target: /usr/src/app/tmp
+        source: agr_generated_files
+        target: /usr/src/agr/tmp
         volume:
           nocopy: true
     environment:
       - PYTHONUNBUFFERED=1
```

Comment:
Don't think you need to set PYTHONUNBUFFERED and use the `-u` switch to the `python` executable (have same effect)